test/test_combinatorica.py

```diff
@@ -1,11 +1,15 @@
 # -*- coding: utf-8 -*-
-from .helper import evaluate, check_evaluation
+from .helper import evaluate, check_evaluation, reset_session
+
+
+def test_00_initialize():
```

Comment:
If you are expecting this to get run first either because of the position of the test or the name of the test, that is fragile and a bad idea.

Reply:
@rocky, I know that, but was the "less imaginative" way I found to do this. I am going to investigate how to do it properly as soon as I have some time for doing it.